src/layman/__init__.py

```diff
@@ -124,6 +124,10 @@
                             if crs_def.internal_srid:
                                 db_util.ensure_srid_definition(crs_def.internal_srid, crs_def.proj4text)
 
+                        logger.info(f'Set wfs_wms_status for layers')
```

Comment:
"Set wfs_wms_status for layers" -> "Change layers PREPARING wfs_wms_status to FAILED"